packages/common/src/css/components/SelectionGrid.css

```diff
@@ -26,7 +20,6 @@
   }
 
   &__image {
-    padding-top: 30%;
     position: relative;
     background-repeat: no-repeat;
```

Comment:
these background properties can probably be removed if we're not using OneImageBackground